XP/install/set-installation-overrides.ps1.example

```diff
@@ -28,36 +25,54 @@ $site.suffix = "dev.local"
 $site.webroot = "C:\inetpub\wwwroot"
 $site.hostName = $json.settings.site.prefix + "." + $json.settings.site.suffix
 $site.habitatHomeSslCertificateName = $site.hostName
-$json.settings.site = $site
 
+# Sitecore Parameters
 $sitecore = $json.settings.sitecore
 $sitecore.siteRoot = Join-Path $site.webRoot -ChildPath $site.hostName
-$json.settings.sitecore = $sitecore
+$sitecore.adminPassword = "b"
+$sitecore.exmCryptographicKey = "0x0000000000000000000000000000000000000000000000000000000000000000"
+$sitecore.exmAuthenticationKey = "0x0000000000000000000000000000000000000000000000000000000000000000"
+$sitecore.telerikEncryptionKey = "PutYourCustomEncryptionKeyHereFrom32To256CharactersLong"
 
 # Solr Parameters
 $solr = $json.settings.solr
 $solr.url = "https://localhost:8983/solr"
 $solr.root = "c:\solr\solr-6.6.2"
 $solr.serviceName = "Solr-6.6.2"
 
-
-$sql = $json.settings.sql
```

Comment:
The `$sql = $json.settings.sql` assignment needs to be added back in. Without this the ps script fails at this point.

Reply:
Absolutely right. Thanks @jeanfrancoislarente for adding it back and merging.